test/testenv/secretutil.go

```diff
@@ -174,3 +181,17 @@ func CheckAdminPasswordViaAPI(deployment *Deployment, podName string, password s
 	}
 	return false
 }
+
+// GetSecretFromInputConf gets give secret from server under given stanza
+func GetSecretFromInputConf(deployment *Deployment, podName string, ns string, configName string, stanza string) (string, string, error) {
+	filePath := "/opt/splunk/etc/apps/splunk_httpinput/local/inputs.conf"
```

Comment:
Based on the deployment, don't we need to change the filePath?
Say, for IDXC, it is /opt/splunk/etc/slave-apps/splunk_httpinput/, whereas for the Standalone it is ../etc/apps/splunk_httpinput

Reply:
Checked in runs. It seems to be the same path

C3
```
tpereira@C02D15YMMD6R splunk-operator % k exec -it splunk-secret-q3s-xmi-idxc-indexer-0 -n secret-q3s bash
kubectl exec [POD] [COMMAND] is DEPRECATED and will be removed in a future version. Use kubectl exec [POD] -- [COMMAND] instead.
[splunk@splunk-secret-q3s-xmi-idxc-indexer-0 splunk]$
[splunk@splunk-secret-q3s-xmi-idxc-indexer-0 splunk]$
[splunk@splunk-secret-q3s-xmi-idxc-indexer-0 splunk]$ cat /opt/splunk/etc/apps/splunk_httpinput/local/inputs.conf
[http]
disabled = 0

[http://splunk_hec_token]
disabled = 0
token = 5B5ACFBC-D003-6008-24FA-79BE3BEC8006
[splunk@splunk-secret-q3s-xmi-idxc-indexer-0 splunk]$
```


S1
```
splunk@splunk-secret-qv3-ey6-standalone-0 splunk]$ cat /opt/splunk/etc/apps/splunk_httpinput/local/inputs.conf
[http]
disabled = 0

[http://splunk_hec_token]
disabled = 0
token = FF0E081D-40A4-C555-A0BB-4411B5B4B45F
[splunk@splunk-secret-qv3-ey6-standalone-0 splunk]$```

Reply:
@sgontla This secret is only checked on Indexer and Standalone.